src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'parser/core/Analyzer';
+import StatisticBox from 'interface/others/StatisticBox';
+import ItemHealingDone from 'interface/others/ItemHealingDone';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber } from 'common/format';
+
+
+
+/**
+ * Simple info graphic that shows what spell did what healing during wotc window, the hps of each spell, and hptc
+ */
+
+class WayOfTheCrane extends Analyzer {
+
+  customMap = null;
+  damageSpell = "";
+  lastTimeStamp = 0;
+  inWotc = false;
+  wotcTime = 0;
```

Comment:
several of these look like internal values that shouldn't be read or used by other modules. you should prefix those with a `_` so other people know.

Reply:
Only if js was a strongly typed language I could just slap a private on it :( but good idea will do 👍

Reply:
yea :(